kolibri/plugins/learn/assets/src/vue/breadcrumbs/index.vue

```diff
@@ -19,7 +19,7 @@
       </template>
 
       <span v-if="pageName === PageNames.EXPLORE_CONTENT">
-        <breadcrumb :linkobject="parentExploreLink"></breadcrumb>
+        <breadcrumb :linkobject="parentExploreLink" :text="('Back')" ></breadcrumb>
```

Comment:
Once this is merged, I'll submit a separate PR to address the current `nav aria-label` issue: since the **Back** button is now inside, the label is read even with this button alone. We'll have to go back to separate `visuallyhidden` spans.

@indirectlylit